.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: E2E Tests
+
+on:
+  workflow_call:
+    inputs:
+      service_endpoint:
+        required: true
+        type: string
+      app_name:
+        required: false
+        type: string
+
+jobs:
+  e2e:
+    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
+    runs-on: ubuntu-latest
+
+    env:
+      BASE_URL: ${{ inputs.service_endpoint }}
+      APP_NAME: ${{ inputs.app_name }}
+
```

Comment:
nit: delete, doesn't look like this is used anywhere. in the "run e2e tests" step, you use inputs.app_name and inputs.service_endpoint, not env.app_name and env.service_endpoint

Reply:
Thanks - looks like it worked:
```
Run make e2e-test APP_NAME=app BASE_URL=http://p-86-app-dev-1797299030.us-east-1.elb.amazonaws.com/
  make e[2](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:2)e-test APP_NAME=app BASE_URL=http://p-86-app-dev-17972990[3](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:3)0.us-east-1.elb.amazonaws.com
  shell: /usr/bin/bash -e {0}

Running [6](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:7) tests using 1 worker
······
  6 passed (8.3s)
  ```